### DISCUSSION TITLE
Spreadsheets: Localization of currency formats

### DISCUSSION THREAD

**Discussion by thob** [standardnotes/forum#2608] [Suggestions]
for some reason I'm on USD but would want to go with EUR. Can I change this somewhere?

**Comment by atmoio** [comment]
Not yet I'm afraid. We want to add better localization here but it will take time.

**Comment by alxciii** [comment]
Would love to get £ into spreadsheets to do my budgets.

**Comment by AKelly404** [comment]
Is this possible yet? I can see a closed issue/discussion but unable to find how to do this so far
